ArchiSteamFarm/IPC/Integration/SwaggerSecurityCriticalAttribute.cs

```diff
@@ -36,10 +35,20 @@ public sealed class SwaggerSecurityCriticalAttribute : CustomSwaggerAttribute {
 	public override void Apply(OpenApiSchema schema) {
 		ArgumentNullException.ThrowIfNull(schema);
 
-		if (schema.Items is { Reference: null }) {
-			schema.Items.AddExtension(ExtensionName, new OpenApiBoolean(true));
-		} else {
-			schema.AddExtension(ExtensionName, new OpenApiBoolean(true));
+		JsonValue value = JsonValue.Create(true);
+
+		if (schema.Items != null) {
+			if (schema.Items is OpenApiSchema items) {
+				items.AddExtension(ExtensionName, new JsonNodeExtension(value));
+			} else if (schema.Items.Extensions != null) {
+				schema.Items.Extensions[ExtensionName] = new JsonNodeExtension(value);
+			} else {
+				throw new InvalidOperationException(nameof(schema.Items));
```

Comment:
Since we're just adding optional extensions, I don't think we should throw in this case and rather do nothing instead. We should do our best to provide additional information, but if we can't for some reason, we shouldn't produce exceptions either in my opinion.

Reply:
`InvalidOperationException` is ASF convention for stuff that we totally don't support and to the best of our knowledge should never happen.

In order to reach the situation you're describing, somebody would have to write plugin that explicitly modifies schema items to a custom object, and on top of that, to an object where they don't initialize the `Extensions` dictionary. Even if somebody would have the rarest edge case that comes to my mind where they needed to implement custom items object that for some reason is not `OpenApiSchema`, we can always tell them to just initialize the `Extensions` for us if they expect our annotations to work with it.

Unless you have some specific case in mind that you can show, I don't consider it supported scenario.